opengeo/core/management/commands/ingest_grib.py

```diff
@@ -0,0 +1,70 @@
+import logging
+import os
+from urllib.parse import urljoin
+
+from bs4 import BeautifulSoup
+import djclick as click
+from rgd.management.commands import _data_helper as helper
+from rgd.models import Collection
+from rgd.utility import url_file_to_local_path
+from rgd_imagery.management.commands import _data_helper as imagery_helper
+from rgd_imagery.models import Image
+
+logger = logging.getLogger(__name__)
+
+# start: 20040302/
+# end: 20040331/
+
+
+@click.command()
+@click.option(
+    '-h',
+    '--host',
+    default='https://www.ncei.noaa.gov/data/global-forecast-system/access/historical/analysis/200403/',
+)
+@click.option('-c', '--collection', default='GRIB')
+@click.option('-x', '--extension', default='.grb')
+def ingest_s3(
+    host: str,
+    collection: str,
+    extension: str,
+) -> None:
+
+    collection, _ = Collection.objects.get_or_create(name=collection)
+
+    for directory in range(20040302, 20040331 + 1):
```

Comment:
This is very specific to the dataset I was asked to ingest. We may want to parameterize this more so that any directory under https://www.ncei.noaa.gov/data/global-forecast-system/access/historical/analysis/ can be ingested